pkg/sql/opt/exec/execbuilder/tests/local/local_test: TestExecBuild_jsonb_path_query failed

pkg/sql/opt/exec/execbuilder/tests/local/local_test.TestExecBuild_jsonb_path_query [failed](https://mesolite.cluster.engflow.com/invocations/default/c3405c1c-7047-4854-af60-ca2bafb08d56?testReportRun=1&testReportShard=46&testReportAttempt=1#targets-Ly9wa2cvc3FsL29wdC9leGVjL2V4ZWNidWlsZGVyL3Rlc3RzL2xvY2FsOmxvY2FsX3Rlc3Q=) on master @ [552cfcb086721820f921ed04c08af7e2df6eabcf](https://github.com/cockroachdb/cockroach/commits/552cfcb086721820f921ed04c08af7e2df6eabcf):


```
=== RUN   TestExecBuild_jsonb_path_query
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestExecBuild_jsonb_path_query2817724068
    test_log_scope.go:82: use -show-logs to present logs inline
    logic.go:3155: 
         
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:151: SELECT message FROM [SHOW KV TRACE FOR SESSION] WHERE message LIKE 'CPut %%' OR message LIKE 'Put %%' OR message LIKE 'Del %%' OR message LIKE 'DelRange %%' OR message LIKE 'ClearRange %%' OR message LIKE 'Get %%' OR message LIKE 'Scan %%' 
        expected:
            Scan /Table/106/2/{???-"a"/Arr/Arr/}, /Table/106/2/{???-"a"/Arr/}
            Scan /Table/106/1/7/0, /Table/106/1/11/0, /Table/106/1/8/0, /Table/106/1/13/0, /Table/106/1/14/0, /Table/106/1/15/0, /Table/106/1/16/0, /Table/106/1/12/0, /Table/106/1/3/0, /Table/106/1/9/0, /Table/106/1/4/0, /Table/106/1/6/0
            
        but found (query options: "kvtrace") :
            Scan /Table/106/2/{???-"a"/Arr/Arr/}, /Table/106/2/{???-"a"/Arr/}
            Scan /Table/106/1/{7-8}, /Table/106/1/1{1-2}, /Table/106/1/{8-9}, /Table/106/1/1{3-4}, /Table/106/1/1{4-5}, /Table/106/1/1{5-6}, /Table/106/1/1{6-7}, /Table/106/1/1{2-3}, /Table/106/1/{3-4}, /Table/106/1/{9-10}, /Table/106/1/{4-5}, /Table/106/1/{6-7}
            
[13:38:30] --- done: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query with config local: 10 tests, 1 failures
    logic.go:4494: 
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:157: error while processing
    logic.go:4494: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:157: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestExecBuild_jsonb_path_query2817724068
--- FAIL: TestExecBuild_jsonb_path_query (0.84s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=46</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_jsonb_path_query.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55053

pkg/sql/opt/exec/execbuilder/tests/local/local_test.TestExecBuild_jsonb_path_query [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20550768?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20550768?buildTab=artifacts#/) on master @ [68413f29f6111f0272fa3df4d38b5ca7d6c6cdad](https://github.com/cockroachdb/cockroach/commits/68413f29f6111f0272fa3df4d38b5ca7d6c6cdad):


```
=== RUN   TestExecBuild_jsonb_path_query
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/ba9c2455be45a144d5be12c1a56b401c/logTestExecBuild_jsonb_path_query1297679123
    test_log_scope.go:82: use -show-logs to present logs inline
    logic.go:3155: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/17251/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:151: SELECT message FROM [SHOW KV TRACE FOR SESSION] WHERE message LIKE 'CPut %%' OR message LIKE 'Put %%' OR message LIKE 'Del %%' OR message LIKE 'DelRange %%' OR message LIKE 'ClearRange %%' OR message LIKE 'Get %%' OR message LIKE 'Scan %%' 
        expected:
            Scan /Table/106/2/{???-"a"/Arr/Arr/}, /Table/106/2/{???-"a"/Arr/}
            Scan /Table/106/1/7/0, /Table/106/1/11/0, /Table/106/1/8/0, /Table/106/1/13/0, /Table/106/1/14/0, /Table/106/1/15/0, /Table/106/1/16/0, /Table/106/1/12/0, /Table/106/1/3/0, /Table/106/1/9/0, /Table/106/1/4/0, /Table/106/1/6/0
            
        but found (query options: "kvtrace") :
            Scan /Table/106/2/{???-"a"/Arr/Arr/}, /Table/106/2/{???-"a"/Arr/}
            Scan /Table/106/1/{7-8}, /Table/106/1/1{1-2}, /Table/106/1/{8-9}, /Table/106/1/1{3-4}, /Table/106/1/1{4-5}, /Table/106/1/1{5-6}, /Table/106/1/1{6-7}, /Table/106/1/1{2-3}, /Table/106/1/{3-4}, /Table/106/1/{9-10}, /Table/106/1/{4-5}, /Table/106/1/{6-7}
            
[17:54:31] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/17251/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query with config local: 10 tests, 1 failures
    logic.go:4494: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/17251/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:157: error while processing
    logic.go:4494: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/17251/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:157: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ba9c2455be45a144d5be12c1a56b401c/logTestExecBuild_jsonb_path_query1297679123
--- FAIL: TestExecBuild_jsonb_path_query (1.33s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_jsonb_path_query.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/opt/exec/execbuilder/tests/local/local_test.TestExecBuild_jsonb_path_query [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20549007?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20549007?buildTab=artifacts#/) on master @ [552cfcb086721820f921ed04c08af7e2df6eabcf](https://github.com/cockroachdb/cockroach/commits/552cfcb086721820f921ed04c08af7e2df6eabcf):


```
=== RUN   TestExecBuild_jsonb_path_query
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/c5452344ee601763d58aa9eed06dc5aa/logTestExecBuild_jsonb_path_query1123457001
    test_log_scope.go:82: use -show-logs to present logs inline
    logic.go:3155: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11881/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:151: SELECT message FROM [SHOW KV TRACE FOR SESSION] WHERE message LIKE 'CPut %%' OR message LIKE 'Put %%' OR message LIKE 'Del %%' OR message LIKE 'DelRange %%' OR message LIKE 'ClearRange %%' OR message LIKE 'Get %%' OR message LIKE 'Scan %%' 
        expected:
            Scan /Table/106/2/{???-"a"/Arr/Arr/}, /Table/106/2/{???-"a"/Arr/}
            Scan /Table/106/1/7/0, /Table/106/1/11/0, /Table/106/1/8/0, /Table/106/1/13/0, /Table/106/1/14/0, /Table/106/1/15/0, /Table/106/1/16/0, /Table/106/1/12/0, /Table/106/1/3/0, /Table/106/1/9/0, /Table/106/1/4/0, /Table/106/1/6/0
            
        but found (query options: "kvtrace") :
            Scan /Table/106/2/{???-"a"/Arr/Arr/}, /Table/106/2/{???-"a"/Arr/}
            Scan /Table/106/1/{7-8}, /Table/106/1/1{1-2}, /Table/106/1/{8-9}, /Table/106/1/1{3-4}, /Table/106/1/1{4-5}, /Table/106/1/1{5-6}, /Table/106/1/1{6-7}, /Table/106/1/1{2-3}, /Table/106/1/{3-4}, /Table/106/1/{9-10}, /Table/106/1/{4-5}, /Table/106/1/{6-7}
            
[19:19:56] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11881/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query with config local: 10 tests, 1 failures
    logic.go:4494: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11881/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:157: error while processing
    logic.go:4494: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/11881/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query:157: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c5452344ee601763d58aa9eed06dc5aa/logTestExecBuild_jsonb_path_query1123457001
--- FAIL: TestExecBuild_jsonb_path_query (1.64s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_jsonb_path_query.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query
+++ b/pkg/sql/opt/exec/execbuilder/testdata/jsonb_path_query
@@ -4,7 +4,8 @@
 statement ok
 CREATE TABLE json_tab (
   a INT PRIMARY KEY,
-  b JSONB
+  b JSONB,
+  FAMILY f1 (a, b)
 )
 
 statement ok

